services/src/messaging/src/main/java/org/openkilda/messaging/command/grpc/DeleteLogicalPortRequest.java

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2019 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.command.grpc;
+
+import org.openkilda.messaging.command.CommandData;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+@Data
+public class DeleteLogicalPortRequest extends CommandData {
+
+    @JsonProperty("address")
+    private String address;
+
+    @JsonProperty("logical_port_number")
+    private Integer logicalPortNumber;
```

Comment:
why not primitive?

Reply:
This is mistake. Fixed in CreateRequest too